### DISCUSSION TITLE
What is the release cadence & process for the Swift extension?

### DISCUSSION THREAD

**Discussion by tristanlabelle** [swiftlang/vscode-swift#568] [Q&A] [answered]
What's the current release cadence and process? Do you use branching, or just a good build of `main`?

Looking at the [version history on the marketplace](https://marketplace.visualstudio.com/items?itemName=sswg.swift-lang), there has been monthly releases since January, but none for the last two months. Meanwhile, there has been no new [GitHub releases](https://github.com/swift-server/vscode-swift/tags) since September 2022.

Thanks!

**Comment by adam-fowler** [comment, marked as answer]
There is no defined cadence. It is based off what fixes, new features have been implemented. There are enough fixes and features implemented such that a new version (1.4) will be released soon. There is a PR with the changelist and version number changes in it, waiting to be approved. 

I haven't been tagging the repo when doing a release recently as one I keep forgetting and two there is a release PR for each version which is a pretty good indicator of which commit was the one for each version.

**Reply by tristanlabelle** [reply]
Thank you! It helps to know
